### ISSUE TITLE
Edit Copy for Pre-Conference Outreach

### ISSUE THREAD

**Issue by minhyeong112** [ubiquity/partnerships#108]
This is the test copy that we made with nReach on our onboarding call.  
Please edit as needed. Once approved, we will use this for outreach.

Links for this Task:
- [Copy](https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit?usp=sharing&ouid=114264483098598390380&rtpof=true&sd=true)

**Comment by 0x4007** [comment]
Please add time estimates when you file new issues so that we can price them quickly.

**Comment by 0x4007** [comment]
I suppose we can self assign everybody who collaborates to split the reward. I don't think I would be able to truly focus on it until after NYE though.

**Comment by sura1-0-1** [comment]
> I suppose we can self assign everybody who collaborates to split the reward. I don't think I would be able to truly focus on it until after NYE though.

I’m available to work on this task!

**Comment by sura1-0-1** [comment]
> ```diff
> ! You do not have the adequate role to start this task (your role is: member). Allowed roles are: collaborator, admin.
> ```

@0x4007  It seems I don’t currently have the required role to begin this task (my role is: member). Could you please adjust my role or let me know how we should proceed?

**Comment by 0x4007** [comment]
This is for core team members only. @gentlementlegen the error message should be changed to be more clear.

**Comment by 0x4007** [comment]
@ubiquityos draft me some automated telegram message outreach sequences. Keep it shorthand, and casual as it is an instant messaging platform - do not write it formal and email style.

The objective is to pique the recipients interest for them to reply. The context is that we're traveling to conferences this year and want to pitch our UbiquityOS solution to manage their remote software development teams more efficiently. Use any context you have on UbiquityOS to try and make a sequence of compelling messages that is likely to result in them replying and subsequently getting us a meeting with them in person during the conference. The recipients attended these conferences in the previous year and we are reaching out to them as we believe they are likely to attend again. 

The plain text of the attached document is as follows:
```
Hi {{FIRST_NAME}}, I am Miguel from Ubiquity DAO. 😁 A few of my community members asked about {{COMPANY}}, are you the right person to talk to in that aspect?

If didn't reply for 1 day

gm🖐, just bumping this up, mind if I share more information?

If didn't reply for 2 days

hey {{FIRST_NAME}}, just sharing our link so that you can see that we're legit😁: ubq.fi

If didn't reply for 2 days

hey, it seems like community building is not a priority right now. Feel free to reach out if that ever changes

If didn't reply for 3 days

{{FIRST_NAME}} ?




Upon receiving reply at any point, use these templates as appropriate:

Awesome! Well we’ve developed UbiquityOS—a sophisticated GitHub bot that's been fine-tuned over the past three years. It streamlines workflows by automating tasks and seamlessly integrates with GitHub to boost productivity. Plus, it offers automated, instant payments upon closing github issues, ensuring smooth and efficient transactions. You can check it out here: [ubq.fi](http://ubq.fi/).
How would it feel to get paid instantly every time your team completes tasks?


Cool! Will {{COMPANY}} be going to (insert soonest conference) by chance? If so, would be awesome to meet!
```

Keep emoji use to a minimum (or none)

**Comment by 0x4007** [comment]
@sshivaditya2019 @gentlementlegen how come ask doesn't work on this repo?

**Comment by gentlementlegen** [comment]
/ask Are you functional in this repository?

**Comment by 0x4007** [comment]
@UbiquityOS draft me some automated telegram message outreach sequences. Keep it shorthand, and casual as it is an instant messaging platform - do not write it formal and email style.

The objective is to pique the recipients interest for them to reply. The context is that we're traveling to conferences this year and want to pitch our UbiquityOS solution to manage their remote software development teams more efficiently. Use any context you have on UbiquityOS to try and make a sequence of compelling messages that is likely to result in them replying and subsequently getting us a meeting with them in person during the conference. The recipients attended these conferences in the previous year and we are reaching out to them as we believe they are likely to attend again. 

The plain text of the attached document is as follows:
```
Hi {{FIRST_NAME}}, I am Miguel from Ubiquity DAO. 😁 A few of my community members asked about {{COMPANY}}, are you the right person to talk to in that aspect?

If didn't reply for 1 day

gm🖐, just bumping this up, mind if I share more information?

If didn't reply for 2 days

hey {{FIRST_NAME}}, just sharing our link so that you can see that we're legit😁: ubq.fi

If didn't reply for 2 days

hey, it seems like community building is not a priority right now. Feel free to reach out if that ever changes

If didn't reply for 3 days

{{FIRST_NAME}} ?




Upon receiving reply at any point, use these templates as appropriate:

Awesome! Well we’ve developed UbiquityOS—a sophisticated GitHub bot that's been fine-tuned over the past three years. It streamlines workflows by automating tasks and seamlessly integrates with GitHub to boost productivity. Plus, it offers automated, instant payments upon closing github issues, ensuring smooth and efficient transactions. You can check it out here: [ubq.fi](http://ubq.fi/).
How would it feel to get paid instantly every time your team completes tasks?


Cool! Will {{COMPANY}} be going to (insert soonest conference) by chance? If so, would be awesome to meet!
```

Keep emoji use to a minimum (or none)

**Comment by sura1-0-1** [comment]
Here’s a refined sequence for your Telegram outreach.

**Initial Message (Day 1)**
Hey {{FIRST_NAME}}, this is Miguel from UbiquityOS. Quick Q—are you still attending [insert conference name]? I’d love to connect while we’re there.

**Follow Up 1 (After 1 Day)**
Hey {{FIRST_NAME}}, just following up on my earlier message. I’d love to share how UbiquityOS can help remote software teams ship faster. Let me know if you’re around to chat at [insert conference].

**Follow Up 2 (After 2 Days)**
Hey, just sharing a quick link so you can check out UbiquityOS: [ubq.fi](http://ubq.fi/). It’s been a game-changer for managing remote teams on GitHub. Any chance we can catch up at [insert conference]?

**Follow Up 3 (After 2 More Days)**
Hey {{FIRST_NAME}}, not sure if this is on your radar right now so I’ll leave it here: [ubq.fi](http://ubq.fi/). Let me know if you’d like to connect sometime or at [insert conference].

**Final Reminder (After 3 More Days)**
{{FIRST_NAME}}, still curious if you’ll be at [insert conference]? Would love to say hi and chat about how we’re helping remote teams move faster.

**If They Reply at Any Point**
**If they’re curious about UbiquityOS:**
Awesome! UbiquityOS is a GitHub add-on that automates workflows, reduces bottlenecks, and even handles instant payments for closed tasks. Teams run smoother, ship faster, and the process feels frictionless. Check it out here: [ubq.fi](http://ubq.fi/).
Does this sound like something your {{COMPANY}} team could benefit from?

**If they confirm they’ll attend the conference:**
Cool! Would be great to meet at [insert conference]. When’s a good time to catch up?

**If they say they’re unsure about the conference:**
Got it! If you do end up going, let me know. In any case, happy to explore how UbiquityOS can help your team boost productivity and cut down on ops overhead.

**If they show interest but mention they may need more info:**
For sure! UbiquityOS is basically like a superpowered GitHub assistant. Streamlines workflows, automates the boring stuff, and ensures teams get paid instantly for task completions. Here's a link for more details: [ubq.fi](http://ubq.fi/).
Would love to deep dive whenever you're ready.

**Comment by sura1-0-1** [comment]
> I suppose we can self assign everybody who collaborates to split the reward. I don't think I would be able to truly focus on it until after NYE though.

https://docs.google.com/document/d/1i-sRzvuqY6eljYLpKiglI7o6oowAkPNT8DuI3quX1K8/edit?usp=sharing

have been trying to refine the email out reach take a look at the docs link.

**Comment by sura1-0-1** [comment]
@0x4007 Let me know things that should be included and improved ? also improve that way I will work on the email reply for Business and LinkedIn Post.

**Comment by minhyeong112** [comment]
new link for the initial drafts (now in the ubiquity drive):
https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit?usp=drive_link&ouid=114264483098598390380&rtpof=true&sd=true

I recommend consolidating all copy there (telegram, email etc.) before marking this complete

**Comment by sura1-0-1** [comment]
[Pre-Conference Outreach Playbook.pdf](https://github.com/user-attachments/files/18324372/Pre-Conference.Outreach.Playbook.pdf)

**Comment by sura1-0-1** [comment]
@minhyeong112 I have included the whole content from the given docs. If there is anything I have to consider changing, let me know.

**Comment by minhyeong112** [comment]
please put in the provided doc link:
https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit?usp=drive_link&ouid=114264483098598390380&rtpof=true&sd=true

**Comment by sura1-0-1** [comment]
> please put in the provided doc link: https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit?usp=drive_link&ouid=114264483098598390380&rtpof=true&sd=true
@minhyeong112 I have updated the document. Take a look at the link above.

**Comment by sura1-0-1** [comment]
@minhyeong112   Is there anything I have to consider modifying or changing ?

**Comment by minhyeong112** [comment]
@0x4007 has final approval of this task

**Comment by 0x4007** [comment]
I left some comments and my primary concern is that the initial messages before receiving a response does not explain anything about what we do and doesn't really give them a reason to want to respond. 

However, if we're messaging from an account with @ana-0k pictures, maybe they would reply just because she's a woman. 

Anyways, looking forward to hearing your rationale. Again, I left comments on the doc that you should review for context.

**Comment by minhyeong112** [comment]
I also added in some comments for your action @sura1-0-1 

> if we're messaging from an account with @ana-0k pictures

@0x4007 I switched over the bio and pic for the nreach-provided to account to Ana's
can also connect her account to instantly double the amount of daily automated outreach

**Comment by 0x4007** [comment]
Maybe best to communicate directly on Telegram since they are on vacation now might not be checking GitHub much.

**Comment by minhyeong112** [comment]
> This is for core team members only. @gentlementlegen the error message should be changed to be more clear.

not sure how I missed this.  will self-assign and make edits now.

**Comment by 0x4007** [comment]
I am also UbiquityOS today

**Comment by minhyeong112** [comment]
> I am also UbiquityOS today

ahaha 

just made edits on the telegram portion based on your comments here (some of your comments were lost on the doc because I erased entire blocks of text):

[Copy](https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit?usp=sharing&ouid=114264483098598390380&rtpof=true&sd=true)

Once you're happy with the TG portion, that should be all we need in my opinion for pre-conference outreach.  Will continuously improve the copy as well as I get feedback from the prospects and as we develop more stuff (more educational materials, calendly link for scheduling, etc)

**Comment by sura1-0-1** [comment]
<img width="541" alt="image" src="https://github.com/user-attachments/assets/802b3f6d-c2c4-4759-b861-7d296b149a93" />
I think I need to request access. Can you give me access to the docs ?

**Comment by 0x4007** [comment]
Tasks in this repo are for core team members only. 

It is actually the main reason why it doesn't appear on https://devpool.directory

**Comment by v4d33m** [comment]
@minhyeong112 ChatGPT prompt ideas to prepare email copy: 


Imagine that you are a copywriter. You've been assigned with a task to prepare a sequence of 3 emails for cold email outreach to {persona} working at {DAO company description} asking if they are going to visit {conference}. The objective is to pique the recipients interest for them to reply. The context is that we're traveling to conferences this year and want to pitch our UbiquityOS solution to manage their remote software development teams more efficiently. Style should be informal yet professional. 

Email 1 should be a meeting request, where we would like to present our solution for their {Needs} and {Frustruations}.
Email 2 should suggest that additional context could help and provide brief information about us that you could find here {UbiquityOS in 1 minute} and why using UbiquityOS would help {Persona} achieve their {Personal Goals}.
Email 3 should be a reference request where we acknowledge that addressee is probably not responding because he/she is not going to visit event this time but we would like to reiterate that UbiquityOS is a solution for {Company Pain Point} that helps to achieve {Business goals} and it would be great if he/she could point us in the right direction.

Every email should be plain text, use personalization placeholders for {{First Name}} and {{Company name}} and fit into a mobile phone screen when read.



{persona}, {DAO company description}, {Needs} and other variables could be taken from #105

**Comment by v4d33m** [comment]
{conferences} could be taken from either #96 or #104 
@0x4007 i think that the final list is pending your approval

**Comment by 0x4007** [comment]
Maybe if I'm lucky, our ChatGPT won't overload its context window:

**Comment by 0x4007** [comment]
/ask Imagine that you are a copywriter. You've been assigned with a task to prepare a sequence of 3 emails for cold email outreach to {persona} working at {DAO company description} asking if they are going to visit {conference}. The objective is to pique the recipients interest for them to reply. The context is that we're traveling to conferences this year and want to pitch our UbiquityOS solution to manage their remote software development teams more efficiently. Style should be informal yet professional. 

Email 1 should be a meeting request, where we would like to present our solution for their {Needs} and {Frustruations}.
Email 2 should suggest that additional context could help and provide brief information about us that you could find here {UbiquityOS in 1 minute} and why using UbiquityOS would help {Persona} achieve their {Personal Goals}.
Email 3 should be a reference request where we acknowledge that addressee is probably not responding because he/she is not going to visit event this time but we would like to reiterate that UbiquityOS is a solution for {Company Pain Point} that helps to achieve {Business goals} and it would be great if he/she could point us in the right direction.

Every email should be plain text, use personalization placeholders for {{First Name}} and {{Company name}} and fit into a mobile phone screen when read.



{persona}, {DAO company description}, {Needs} and other variables could be taken from #105

**Comment by 0x4007** [comment]
@gentlementlegen I think that `development` needs to be merged to `main` because I remember that error was fixed. 

For context to minhyeong112 and v4d33m, we run the development instances of the plugins on the other two orgs @ubiquity-os and @ubiquity-os-marketplace. We stick with production on @ubiquity

**Comment by gentlementlegen** [comment]
@0x4007 Merged just now: https://github.com/ubiquity-os-marketplace/command-ask/pull/49

**Comment by 0x4007** [comment]
/ask Imagine that you are a copywriter. You've been assigned with a task to prepare a sequence of 3 emails for cold email outreach to {persona} working at {DAO company description} asking if they are going to visit {conference}. The objective is to pique the recipients interest for them to reply. The context is that we're traveling to conferences this year and want to pitch our UbiquityOS solution to manage their remote software development teams more efficiently. Style should be informal yet professional. 

Email 1 should be a meeting request, where we would like to present our solution for their {Needs} and {Frustruations}.
Email 2 should suggest that additional context could help and provide brief information about us that you could find here {UbiquityOS in 1 minute} and why using UbiquityOS would help {Persona} achieve their {Personal Goals}.
Email 3 should be a reference request where we acknowledge that addressee is probably not responding because he/she is not going to visit event this time but we would like to reiterate that UbiquityOS is a solution for {Company Pain Point} that helps to achieve {Business goals} and it would be great if he/she could point us in the right direction.

Every email should be plain text, use personalization placeholders for {{First Name}} and {{Company name}} and fit into a mobile phone screen when read.



{persona}, {DAO company description}, {Needs} and other variables could be taken from #105

**Comment by 0x4007** [comment]
/ask Imagine that you are a copywriter. You've been assigned with a task to prepare a sequence of 3 emails for cold email outreach to {persona} working at {DAO company description} asking if they are going to visit {conference}. The objective is to pique the recipients interest for them to reply. The context is that we're traveling to conferences this year and want to pitch our UbiquityOS solution to manage their remote software development teams more efficiently. Style should be informal yet professional. 

Email 1 should be a meeting request, where we would like to present our solution for their {Needs} and {Frustruations}.
Email 2 should suggest that additional context could help and provide brief information about us that you could find here {UbiquityOS in 1 minute} and why using UbiquityOS would help {Persona} achieve their {Personal Goals}.
Email 3 should be a reference request where we acknowledge that addressee is probably not responding because he/she is not going to visit event this time but we would like to reiterate that UbiquityOS is a solution for {Company Pain Point} that helps to achieve {Business goals} and it would be great if he/she could point us in the right direction.

Every email should be plain text, use personalization placeholders for {{First Name}} and {{Company name}} and fit into a mobile phone screen when read.



{persona}, {DAO company description}, {Needs} and other variables could be taken from #105 

---

use this spreadsheet's contents

```csv
ICP - the perfect company or customer you want to target for your business,,,,,,,,,
,,,,,,,,,
,,,,,,,,,
,Company Fit,,,,,,,,
,,Industry / Type of Company,Location / Geo,Company Size,Budget,Pain Points,Business Goals,Technologies,Attributes
,Description / Questions,Which industries are most likely to need your product/service?,Where are these companies primarily located?,What size companies are you targeting?,Can they afford your offerings?,What specific problems do you solve for them?,How can you help them achieve their goals?,What technologies do they currently use or need?,What other attributes define our ideal customer?
,DAO ,"Blue-chip DAOs, ETH ecosystem only ",worldwide 100% remote ,"top-tier DAOs: tens to hundreds of thousands of nodes/validators, tens of millions of transactions, that kind of size ","we start with paid PoCs, covering set-up   fees neglible to DAO  development budget. Then we switch to transactional cost. ","Excessive hierarchy, it's difficult to manage DAO contributors/builders: You’ll need managers that are available during every time zone and also offer rewards appropriately according to work inputs, In a true DAO structure, there is a spectrum of how involved any individual contributor is. They can also be located anywhere in the world. Some may post a single comment and some may work 80+ hour weeks on the project, and everything in between. Backlog buildup, slow project delivery. Read this blog article by dYdX founder and CEO: https://dydx.exchange/blog/regenesis for more context on pain points of DAOs reaching certain level of maturity. ","Automating routine tasks, decreasing management efforts and operational costs","GitHub, ETH ",10+ contributors on GitHub
,VC,Crypto-friendly top-tier VCs for strategic financing,worldwide crypto-friendly ,$100M+ of dry powder,$10-25M ,investors are looking for credible projects that align with their missions and focus ,invest in Round A startups,"Related to crypto, e.g. ConsenSys or Paradigm or tech - e.g. M12 (aka GitHub fund) ",
,,,,,,,,,
,,,,,,,,,
,Persona Fit,,,,,,,,
,,Role,Department,Influence Level,Needs,Frustrations,Goals,Decision Criteria,Content Preference
,Description / Questions,"Identify roles most involved in evaluation and decision-making for your product/service (e.g., C-Level, VP, Director).","Departments directly benefiting from or influencing the purchase (e.g., IT, Operations, Finance).",What is their level of influence or decision-making authority?,Specific needs addressed by your product/service.,Challenges they aim to solve.,Their personal or departmental goals.,"Key attributes that align with their priorities (Price, Features, ROI).","The type of content that engages them (Case Studies, Demos, Webinars)."
,DAO Community/DevRel Managers,"Community managers, DevRels, Engagement managers","developers, opensource contributors","Influence Level, internal champions",Have more devs working on the project with higher motivation ,build DAO that is loved by builders ,contributor/developer/community satisfaction ,Instant gratification on task completion,Case study 
,DAO managers/team leads,"Team leads, PMs, tech management ","developers, opensource contributors",PoC Implementation,"Get rid of mundane task management, deliver more in less time",Distributed team is difficult to manage and requires new approach,"roadmap delivery, time-to-market",Automatic assigments,Demo 
,DAO Founders/C-level,"CTO, CEO, CFO, Founders ",Org-wide ,Purchasing Level ,Cut the middleman and management costs ,"Get back to ""startup"" mode and bring ""A"" into their DAO",operational efficiency (being cash-positive),ROI,Personal stories of other high-profile community members 
,VC Scout,"Scout, analyst, investment associate, investment team ",scouting,"Influence Level, internal champions",finding exciting projects with good traction and promising ROI,project won't pass scoring / due diligence ,get projects that would pass their scoring and align well with their goals  ,"financials, team, tech, risks","pitch deck, data room "
,VC Partner,"MD, GP, Partner, Investor",VC firm-wide,decision making ,finding projects worth risking their money ,losses and write-offs ,"save and multiply capital, satisfy LPs ","captable, financials, team, tech, risks",Scoring / Invest-memos from other VCs
```

Also refer to the powerpoint contents
```
Ubiquity OS outreach strategy
2025

Value Prop / Framework
Company Type
Context/Use Case
Problem
Persona
The industry, company size, business model, etc.
The task or goal the persona is trying to accomplish
The issues the persona is running into because of the current solution
The department or “Champion” we’re selling Ubiquity to
Alternative
The way the persona is currently going about accomplishing this
Capability
Feature/
Category
Benefit
The “Unlock” Ubiquity OS enables, that solves the use case without the problem of the alternative
The specific features, that make this unlock possible
The outcome of using Ubiquity instead of the alternative solution
Some
one
Working in
Doing
Which leads to
Using
Is solved by
Powered by
Which leads to
Decide if Alternative or Context has a better link with problem
Opposites

ICP – Org level
Company Fit
 
Industry / Type of Company
Location / Geo
Company Size
Budget
Pain Points
Business Goals
Technologies
Attributes
Description/Question
Which industries are most likely to need our product?
Where are these companies primarily located?
What size companies are you targeting?
Can they afford your offerings?
What specific problems do you solve for them?
How can you help them achieve their goals?
What technologies do they currently use or need?
What other attributes define our ideal customer?
DAO 
Blue-chip DAOs, ETH ecosystem only 
Worldwide 100% remote 
Top-tier DAOs: tens to hundreds of thousands of nodes/validators, tens of millions of transactions, that kind of size 
We start with paid PoCs, covering set-up   fees neglible to DAO  development budget. Then we switch to transactional cost. On the other hand -  
Excessive hierarchy, lack of motivation for DAO contributors/builders, backlog buildup, slow project delivery. Read this blog article by dYdX founder
Faster time-to-market, with less management efforts 
GitHub, ETH 
10+ contributors on GitHub
VC
Crypto-friendly top-tier VCs for strategic financing
US-based 
$100M+ of dry powder
$10-25M 
investors are looking for credible projects that align with their missions and focus 
invest in Round A startups
Related to crypto, e.g. ConsenSys or Paradigm or tech - e.g. M12 (aka GitHub fund) 
 
3

ICP – Persona level (DAOs)
Persona Fit
 
Role
Department
Influence Level
Needs
Frustrations
Goals
Decision Criteria
Content Preference
Description / Questions
Identify roles most involved in evaluation and decision-making for your product/service
Departments directly benefiting from or influencing the purchase 
What is their level of influence or decision-making authority?
Specific needs addressed by your product/service.
Challenges they aim to solve.
Their personal or departmental goals.
Key attributes that align with their priorities 
The type of content that engages them 
DAO Community/DevRel Managers
Community managers, DevRels, Engagement managers
developers, opensource contributors
Influence Level, internal champions
Have more devs working on the project with higher motivation 
build DAO that is loved by builders 
contributor/developer/community satisfaction 
Feature: Instant gratification on task completion
Social media, Case studies
DAO managers/team leads
Team leads, PMs, tech management 
developers, opensource contributors
PoC Implementation
Get rid of mundane task management, deliver more in less time
Distributed team is difficult to manage and requires new approach
roadmap delivery, time-to-market
Feature: Automatic assigments
Demos and technical papers
DAO Founders/C-level
CTO, CEO, CFO, Founders 
Org-wide 
Purchasing Level 
Cut the middleman and management costs 
Get back to "startup" mode and bring "A" into their DAO
operational efficiency (being cash-positive)
ROI
Personal stories of other high-profile community members 
4

Prospecting
DAOs: 
Initial list 
ETH ecosystem events – web parsing (sponsors and speakers) 
GitHub 
VCs:
Initial list 
Crunchbase 
Cryptorank 

Enrichment – Clay (LI, emails, phone numbers), nReach (telegram) 
5

Channels
LinkedIn
Campaigns to build up personal networks for Alex and Miguel prior email campaigns
Social selling / outreach campaigns following email campaigns 
Social selling / posting 
Email campaign
4 touches with soft CTA 
Telegram campaigns via nReach 
Telegram by mobile number 




Tools
CRM
Prospecting
 telegram
Prospecting& enriching 
linkedin campaigns
email campaigns
VC lists
 telegramcampaigns
Events
scrape
scrape
csv
csv
integration
integration
7

Campaigns
Static DAO/VC campaign: 
1 persona = 1 campaign 
Heyreach “connection builder” campaign (2 weeks prior to email campaign) 
Email campaign to all new connections (4 emails over 2 weeks)  
Heyreach micro-campaign to 1st degree connections, launched in parallel to Email campaign 
Email campaign to 2nd degree connections (4 emails over 2 weeks) 
Telegram campaign in nReach (in parallel) 
ESP matching for Email Campaigns 
Day
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
x
x
x
x
x
x
x
x
x
x
8

Campaigns
Event campaign: 
1 event = 1 campaign 
Heyreach “connection builder” campaign (3 weeks prior to event) 
Email campaign to all new connections (3 emails over 2 weeks)  
Heyreach micro-campaign to 1st degree connections, launched in parallel to Email campaign 
Email campaign to 2nd degree connections (3 emails over 2 weeks) 
Telegram campaign in nReach (in parallel) 
ESP matching for Email Campaigns 
Day
-14
-13
-12
-11
-10
-9
-8
-7
-6
-5
-4
-3
-2
-1
0
x
x
x
x
x
x
x
x
x
x
x
x
9

Messaging
Events: https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit#heading=h.n19grevn90yq


10

```

**Comment by 0x4007** [comment]
/ask now write a sequence for every role/persona

Persona Fit,,,,,,,,
,,Role,Department,Influence Level,Needs,Frustrations,Goals,Decision Criteria,Content Preference
,Description / Questions,"Identify roles most involved in evaluation and decision-making for your product/service (e.g., C-Level, VP, Director).","Departments directly benefiting from or influencing the purchase (e.g., IT, Operations, Finance).",What is their level of influence or decision-making authority?,Specific needs addressed by your product/service.,Challenges they aim to solve.,Their personal or departmental goals.,"Key attributes that align with their priorities (Price, Features, ROI).","The type of content that engages them (Case Studies, Demos, Webinars)."
,DAO Community/DevRel Managers,"Community managers, DevRels, Engagement managers","developers, opensource contributors","Influence Level, internal champions",Have more devs working on the project with higher motivation ,build DAO that is loved by builders ,contributor/developer/community satisfaction ,Instant gratification on task completion,Case study 
,DAO managers/team leads,"Team leads, PMs, tech management ","developers, opensource contributors",PoC Implementation,"Get rid of mundane task management, deliver more in less time",Distributed team is difficult to manage and requires new approach,"roadmap delivery, time-to-market",Automatic assigments,Demo 
,DAO Founders/C-level,"CTO, CEO, CFO, Founders ",Org-wide ,Purchasing Level ,Cut the middleman and management costs ,"Get back to ""startup"" mode and bring ""A"" into their DAO",operational efficiency (being cash-positive),ROI,Personal stories of other high-profile community members 
,VC Scout,"Scout, analyst, investment associate, investment team ",scouting,"Influence Level, internal champions",finding exciting projects with good traction and promising ROI,project won't pass scoring / due diligence ,get projects that would pass their scoring and align well with their goals  ,"financials, team, tech, risks","pitch deck, data room "
,VC Partner,"MD, GP, Partner, Investor",VC firm-wide,decision making ,finding projects worth risking their money ,losses and write-offs ,"save and multiply capital, satisfy LPs ","captable, financials, team, tech, risks",Scoring / Invest-memos from other VCs

**Comment by 0x4007** [comment]
@shiv810 perhaps we can implement a reranker to get better results.

**Comment by 0x4007** [comment]
/ask write out the full email sequences of each persona. Leave out your notes.

**Comment by 0x4007** [comment]
I think the output is not that useful especially for VCs its hallucinated.

**Comment by v4d33m** [comment]
to be honest, i'd just write the copy myself. i'm not very impressed with the results either. those "hope my email finds you well" AI-generated emails not gonna work

**Comment by minhyeong112** [comment]
- Ok I've updated the Telegram/Linkedin copy.  There is an automated portion, and a manual portion to be used as templates for most common replies at my discretion once prospects answer.

- For email, I recommend using very similar copy, but without the links in the automated portion, and with slight modifications for email format (signature block, 2 line breaks after the greeting, etc).  I can build it quickly once the tg/li portion is approved by you guys! @0x4007 @v4d33m  

Here is the centralized "copy" link again for convenience:
[Copy](https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit?usp=sharing&ouid=114264483098598390380&rtpof=true&sd=true)

**Comment by minhyeong112** [comment]
@0x4007 I've modified according to your comments.  Plz let me know what you think now.
In the interest of time, I'll go ahead and launch a tg campaign for the HK consensus attendees from nreach now.  Will adjust the copy with any modifications as we go.

**Comment by minhyeong112** [comment]
- just launched a new tg campaign on nreach to all 78 hk consensus attendees with the updated copy
- I actually replaced the link to the educational doc with these screenshots since links are more "threatening" in the web3 space

<img width="389" alt="Image" src="https://github.com/user-attachments/assets/ec92a106-4bd1-4cda-a0a8-77054dd6cb48" />

<img width="386" alt="Image" src="https://github.com/user-attachments/assets/ae420790-fae5-45a1-b753-5d1db4721a0c" />

- I'll reply as Ana with the goal of getting them interested and getting them to book a spot here: [Ubiquity DAO Calendly](https://calendly.com/ubiquitydao)
- @0x4007 I also created a new chatroom on tg for continuous improvements to the copy.  when I don't know exactly what to reply, I'll post a screenshot there for your/someone's feedback and I'll continuously improve the "manual" section of the copy google doc so that we will always have a list of common replies to common questions/comments from prospects and a template of how to respond

Here is the centralized "copy" link again for convenience:
[Copy](https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit?usp=sharing&ouid=114264483098598390380&rtpof=true&sd=true)

**Comment by 0x4007** [comment]
> - just launched a new tg campaign on nreach to all 78 hk consensus attendees with the updated copy

It's too bad there's so little results 

> - I actually replaced the link to the educational doc with these screenshots since links are more "threatening" in the web3 space

I think it may be better to paste in the text directly? I'm not sure. Nobody likes long messages, but reading the screenshots can be difficult on mobile? 

Maybe we should have a larger font as most people are probably reading from mobile? 

> - I'll reply as Ana with the goal of getting them interested and getting them to book a spot here: [Ubiquity DAO Calendly](https://calendly.com/ubiquitydao)

Sounds good

> - @0x4007 I also created a new chatroom on tg for continuous improvements to the copy.  when I don't know exactly what to reply, I'll post a screenshot there for your/someone's feedback and I'll continuously improve the "manual" section of the copy google doc so that we will always have a list of common replies to common questions/comments from prospects and a template of how to respond


This is a good idea and currently we have the Telegram group chat plug-in disabled because we need to reconfigure some stuff, but this would've been a perfect application of it. Let's proceed with your custom group chat but please post the link in the task thread next time (here) so that in the future UbiquityOS can parse the contents. 

> 
> Here is the centralized "copy" link again for convenience:
> [Copy](https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit?usp=sharing&ouid=114264483098598390380&rtpof=true&sd=true)

**Comment by minhyeong112** [comment]
> I think it may be better to paste in the text directly

updated to this after getting confirmation from nReach that links are ok:

hey {{FIRST_NAME}}, just sharing a bit more about our product and how it’s going to make life so much better for you guys😁

UbiquityOS in 3 Minutes (link and some screenshots in case you prefer that)
https://docs.google.com/document/d/170WbrPe0OH0hzZJcodXo-CsF3QQQrDGlp1_EWqMEjBU/edit?usp=sharing

> post the link in the task thread

would be cool to have a persistent link to any docs/chats that are relevant to a particular task/issue/thread maybe as a custom label?
but for now...

[Copy](https://docs.google.com/document/d/1NZzqT5e0EX6ugxJfl5IquNZikpSyvSmU/edit?usp=sharing&ouid=114264483098598390380&rtpof=true&sd=true)
[Copy Improvements Chat](https://t.me/+K0sWeDGFhI42OWFl)

**Comment by 0x4007** [comment]
> would be cool to have a persistent link to any docs/chats that are relevant to a particular task/issue/thread maybe as a custom label?

What do you mean by this?

**Comment by minhyeong112** [comment]
> What do you mean by this?

Just tried adding the telegram chat link as a GitHub label to this issue's comment thread.  Basically, it would be cool to have these 2 links somewhere easily accessible in this issue comment thread.  But maybe the best way is to edit the opening comment all the way at the top

**Comment by 0x4007** [comment]
Yes I think that editing the specification makes the most sense. We don't want extra labels since it adds clutter, and it isn't even clickable as a link.

**Comment by minhyeong112** [comment]
Ok great! Specification updated with link. I will use the format in the specification for this issue in all future tasks that have associated google doc links.

I am currently using the link for replies to the nreach tg campaign and will edit it iteratively as new situations come about.

Unless there is anything else needed, requesting approval for this task.

**Comment by 0x4007** [comment]
https://github.com/ubiquity/business-development/issues/112#issuecomment-2599905798

This copy is good perhaps it can be incorporated here?

**Comment by 0x4007** [comment]
@gentlementlegen i thought slash commands are filtered out from scoring

**Comment by gentlementlegen** [comment]
@0x4007 yes they are supposed to, I didn't see any slah command in the scoring?

**Comment by 0x4007** [comment]
Check my scoring. https://github.com/ubiquity/business-development/issues/108#issuecomment-2585165654

**Comment by gentlementlegen** [comment]
@0x4007 It seems that the `/ask` got stripped away and the following was evaluated. This is probably because we never had multiline commands before, will open a ticket.

**Comment by v4d33m** [comment]
one more thing @0x4007 

<img width="840" alt="Image" src="https://github.com/user-attachments/assets/1a7b56c9-8278-43cd-a677-e7305087bb00" />

**Comment by 0x4007** [comment]
Let me handle this funding situation. Thanks for letting the financier know!

**Comment by minhyeong112** [comment]
> [#112 (comment)](https://github.com/ubiquity/business-development/issues/112#issuecomment-2599905798)
> 
> This copy is good perhaps it can be incorporated here?

replying on the #112 comment thread
